docs/generate.py

```diff
@@ -7,7 +7,7 @@
 Requires a local installation of `tensorflow_docs`:
 
 ```
-pip install git+https://github.com/tensorflow/docs
+pip install git+https://github.com/arig23498/docs@wandb-docs
```

Comment:
@vanpelt confirm pip install from personal github.

Reply:
No this will not work.  @ariG23498 if we need to use a forked version of tensorflow_docs we need to fork it ourselves.  What changes did you make?

Reply:
Basic templating changes to make it work with `gitbook`.

Reply:
@farizrahman4u could you work with @ariG23498 today to look at the changes and potentially fork the tensorflow/docs inside a new repo wandb/docgen

Reply:
Done